site/Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM gcr.io/{{ global.project }}/debian:9.5
+FROM {{ hail_ubuntu_image.image }}
 
-RUN apt-get update && \
-    apt-get install -y nginx curl python-minimal && \
-    rm -rf /var/lib/apt/lists/*
+RUN hail-apt-get-install \
+  nginx \
+  curl \
+  python-minimal \
+  libnginx-mod-http-subs-filter
```

Comment:
Where did this one come from?

Reply:
Ubuntu nginx does not include this nginx module which we use. Debian nginx included it by default.